site/content/en/docs/tutorials/nvidia.md

```diff
@@ -46,7 +46,7 @@ date: 2018-01-02
   minikube delete
   ```
   This will make sure minikube does any required setup or addon installs now that the nvidia runtime is available.
-  
+
```

Comment:
I can split the unrelated whitespace changes, they were created by my editor, cleaning up whitespaces automatically.